Dima/text/fizz_buzz/fizz_buzz.py

```diff
@@ -0,0 +1,28 @@
+def check_fizzbuzz(number):
+    if number % 3 == 0 and number % 5 == 0:
+        return "FizzBuzz"
+    elif number % 3 == 0:
+        return "Fizz"
+    elif number % 5 == 0:
+        return "Buzz"
+    else:
+        return number
```

Comment:
Still there is no contract in the function. Should return 'Fizz', 'Buzz' or 'FizBuzz'.